### ISSUE TITLE
[Question] Centre View

### ISSUE THREAD

**Issue by idr4n** [HappenApps/Quiver#1092]
In editors such as Sublime or VSCode, I can set an option to centre the view (sometimes called typewriter mode), something that Ulysses as well has as default.

Is there any chance to implement this option in Quiver?

Thanks in advance!

**Comment by ylian** [comment]
Yeah, there is an option in Preferences > General > Scroll past end. It lets you scroll past the last line, but there is no typewriter mode in Quiver.
